app/models/allocation.rb

```diff
@@ -77,32 +77,6 @@ def override_reasons
     JSON.parse(self[:override_reasons]) if self[:override_reasons].present?
   end
 
-  def deallocate_offender(movement_type)
```

Comment:
Whilst the `deallocate_offender` method has moved to the `AllocationService`, the related methods `deallocate_primary_pom` and `deallocate_secondary_pom` have stayed in the `Allocation` model.

It feels like they belong together, since they all do very similar things. It could be a bit confusing to have them split across two places.